flocker/acceptance/scripts/exitinghttp.py

```diff
@@ -3,16 +3,17 @@
 """
 
 from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
-import os
 
 
 class Handler(BaseHTTPRequestHandler):
     def do_GET(s):
         s.send_response(200)
+        s.send_header("content-length", "2")
         s.end_headers()
         s.wfile.write(b"hi")
+        s.wfile.flush()
         s.wfile.close()
-        os._exit(1)
+        raise SystemExit()
```

Comment:
I'm pretty sure @lukemarsden and I tried `SystemExit` and found a problem with it. I might just check that `HTTPServer.serve_forever` doesn't catch it.